WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -175,6 +178,10 @@ public CoinJoinStateViewModel(UiContext uiContext, IWalletModel wallet, WalletSe
 				await mainViewModel.SettingsPage.ActivateCoordinatorTab();
 			}
 		});
+
+		CoordinatorHelpCommand = ReactiveCommand.CreateFromTask(() => UiContext.FileSystem.OpenBrowserAsync("https://www.google.com"));
```

Comment:
open google??

Reply:
why is it merged with this???

Reply:
Arf thanks it's because this command exists in 2 places